exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/ResultRow.kt

```diff
@@ -80,6 +80,7 @@ class ResultRow(
             raw == NotInitializedValue -> error("$expression is not initialized yet")
             expression is ExpressionAlias<T> && expression.delegate is ExpressionWithColumnType<T> -> expression.delegate.columnType.valueFromDB(raw)
             expression is ExpressionWithColumnType<T> -> expression.columnType.valueFromDB(raw)
+            expression is ExpressionAlias<T> -> rawToColumnValue(raw, expression.delegate)
```

Comment:
Looks like it can be simplified.

In the case `expression` is `ExpressionAlias` it will rely on `delegate.columnType.valueFromDB(raw)` if it's possible or fall into recursion with delegate (`rawToColumnValue(raw, expression.delegate)`).

but if `delegate` is `ExpressionWithColumnType` it could be handled after recursion also, so looks like the line `expression is ExpressionAlias<T> && expression.delegate is ExpressionWithColumnType<T> ->` could be safely removed.